src/shared/components/AppBar/AppBar.tsx

```diff
@@ -0,0 +1,34 @@
+import Router from 'next/router';
+
+import { AppBarProps } from './AppBar.types';
+import * as S from './AppBar.styles';
+import Icon from '../Icon';
+
+const AppBar = ({
+  leftIconName = 'ArrowLeft',
+  onLeftIconClick = () => {
+    Router.back();
+  },
+  rightIconName,
+  onRightIconClick,
+  title = '',
+  titleOrient = 'center',
+}: AppBarProps) => {
+  return (
+    <S.Wrapper>
+      {leftIconName && (
+        <S.IconButton onClick={onLeftIconClick}>
+          <Icon name={leftIconName} width="100%" height="100%" />
+        </S.IconButton>
+      )}
```

Comment:
버튼 태그의 type 디폴트 값이 submit 이기 때문에 type='button'을 주어야 할 것 같습니다!
(rightIcon도 마찬가지)

```suggestion
      {leftIconName && (
        <S.IconButton type="button" onClick={onLeftIconClick}>
          <Icon name={leftIconName} width="100%" height="100%" />
        </S.IconButton>
      )}
```

Reply:
224a023a4f1e337c62a43f1d3da30ea9995bfcd2